kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/commands/JSONSchemaCommand.java

```diff
@@ -65,6 +65,15 @@ public JSONSchemaCommand(Logger console) {
       description="Minimize schema file size by eliminating white space")
   boolean minimize=false;
 
+  @edu.umd.cs.findbugs.annotations.SuppressWarnings(
+      value="UWF_NULL_FIELD",
+      justification = "Field set by JCommander")
+  @Parameter(names={"-n", "--num-records"},
+             description="Number of records to derive schema from",
+             arity=1)
+  Integer numRecords = 10;  // By default we look at first 10 records
```

Comment:
Just because I am new to JCommander, and I think their example when it came to integers was using the Object Integer instead of the primitive type.  I changed it to int, and validated it works fine. So I'll checkin that adjustment.